### ISSUE TITLE
No access via iAmpache on iOS (Unauthorized access attempt to API)

### ISSUE THREAD

**Issue by Khymick** [ampache/ampache#174]
I am trying to connect to Ampache 3.6-alpha6+FUTURE from iAmpache on a iOS device. When I type in the URL (I am using https) and the correct credentials (login via website with same username+password works perfectly), I get the following error in the log:

2014-05-11 15:44:50 [ampache] (i18n) -> Setting locale to de_DE.UTF-8 
2014-05-11 15:44:50 [ampache] (i18n) -> gettext is native 
2014-05-11 15:44:50 [ampache] (Access Denied) -> Unauthorized access attempt to API 

How can this be fixed?

**Comment by Afterster** [comment]
Issue is client side, closing this.

**Comment by Khymick** [comment]
Hi,

 

don´t get me wrong, I don´t want to sound bad in any way, but your answer perfectly fits in my thoughts about the development of the app: Closing an issue with "is on client side" without any further comments shows clearly that you are not interested in helping fixing issues - it would be very helpful it you had at least said what is wrong on client side. I still can´t use your app at all and I will certainly comment it on the store after this - and of course delete it from the phone.

 

Gesendet: Donnerstag, 22. Januar 2015 um 08:46 Uhr
Von: Afterster <notifications@github.com>
An: ampache/ampache <ampache@noreply.github.com>
Cc: Khymick <dennismail@gmx.net>
Betreff: Re: [ampache] No access via iAmpache on iOS (Unauthorized access attempt to API) (#174)


Issue is client side, closing this.

**Comment by Afterster** [comment]
Hi @Khymick. I believe you mix up people and apps here :)
I closed the issue because as discussed before, it is identified *client side* and this repository is about Ampache (server). We are not able to provide a solution then.
The client itself is not developed by us (Ampache maintainers) but by Ryan who doesn't manage bugs here.

**Comment by Khymick** [comment]
Hi,

 

I am really sorry - yes, I did mix up people. Anyhow, My Ampache Server is still runnig very good and everything works fine. So no problem on this side.

What concerns the app - well, everyone need time to develop applications and to fix problems, but for AmpacheX, I gave up: After all this time, it still doesn´t work and feedbacks are rare. So I think that the app is - at least for me - unusable.

 

What concerns your work: No problem there - keep on and thanks for it!

 

Gesendet: Donnerstag, 22. Januar 2015 um 10:51 Uhr
Von: Afterster <notifications@github.com>
An: ampache/ampache <ampache@noreply.github.com>
Cc: Khymick <dennismail@gmx.net>
Betreff: Re: [ampache] No access via iAmpache on iOS (Unauthorized access attempt to API) (#174)


Hi @Khymick. I believe you mix up people and apps here :)
I closed the issue because as discussed before, it is identified client side and this repository is about Ampache (server). We are not able to provide a solution then.
The client itself is not developed by us (Ampache maintainers) but by Ryan who doesn't manage bugs here.
